### PR TITLE
Lesson 5 Assignment files

### PR DESCRIPTION
**Pull request by zaandrew**

### CODE REVIEW

File: students/andrey_zkn/lesson05/mailroom3.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    return message
+
+                                      
+def send_thank_you():
+    mail_to = input ("Enter the full name of a donor or 'list' for current donors ")
+    while mail_to.lower() == "list":
+        for name in donors:
+            print(name)
+        mail_to = input ("Please enter a  full name of a donor ")        
+    amount = float(input ("Enter the donation amount "))
```
Comment (natasha-aleksandrova) [inline students/andrey_zkn/lesson05/mailroom3.py:38]: should have exception handling here in a case user enters non-number value

File: students/andrey_zkn/lesson05/mailroom3.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+    
+def main():
+    options_dict = {1: send_thank_you, 2: create_report, 3: send_letters_to_all, 4: quit_now }
+    while True:
+        option = int(menu_options())
+        if option not in options_dict:
+            print("Not a valid option. Please select one of the available options!")
+        else: 
+           options_dict.get(option)()
```
Comment (natasha-aleksandrova) [inline students/andrey_zkn/lesson05/mailroom3.py:84]: this should also have some exception handling (hint: catch KeyError)

File: students/andrey_zkn/lesson05/mailroom3.py (hunk 1 above)
Comment (zaandrew) [reply students/andrey_zkn/lesson05/mailroom3.py:38]: Added this exception handling here. Thank you for the hint. The mailroom3.py file has been resubmitted. Please see my new pull request.

File: students/andrey_zkn/lesson05/mailroom3.py (hunk 2 above)
Comment (zaandrew) [reply students/andrey_zkn/lesson05/mailroom3.py:84]: Added this exception handling here. Thank you for the hint. The mailroom3.py file has been resubmitted. Please see my new pull request.

**Comment by natasha-aleksandrova** [comment]
great! we will take another look 🙂

**Pull request merged**